CHANGELOG.md

```diff
@@ -1,108 +1,91 @@
-| Bug   | Remove server chunk generation from webpack                                                                                             | [3608][]          |
-| Bug   | \[Issue] Improve Gallery/AddToCartButton.js                                                                                             | [3573][]          |
-| Bug   | \[bug]: Breadcrumbs 'Shop' link redirects to '/shopnull'                                                                                | [3603][]          |
-| Bug   | Newsletter form displays two errors                                                                                                     | [3584][]          |
-| Bug   | Home page tabbing leads to page transition                                                                                              | [3625][]          |
-| Bug   | Browser Reload on any Category load all products from default category                                                                  | [3655][], [18][]  |
+
+## Highlights  
+  
+-   Added shimmer loaders for Dynamic blocks to avoid layout shift. Shimmer loaders are only rendered if `minHeight` is specified on the Row containing the Dynamic block. This implementation gives you a choice: You can set the Row's `minHeight` to render the shimmer or remove the `minHeight` to remove the shimmer. For Dynamic blocks that rarely render, we recommend removing the `minHeight` property from the Rows containing those Dynamic blocks. — [3713][]
+-   Added the new `PersonalizedContentSampleDataVenia` module that contains customer segments, dynamic blocks, and cart rule promotion sample data for personalized content. — [19][],[2][],[3670][]
+-   You can now use ReCaptcha V3 on the Braintree payment form and the Place Order form. — [3691][],[24][]
+-   reCAPTCHA V3 validation is now available on the following customer forms:
+    - Customer Login
+    - Customer account creation
+    - Customer information change (password) — [3702][]
+-   reCaptcha has been enabled for the Forgot Password form. — [3604][],[12][],[20][]
+-   In the Link component, `prefetchType` property has been renamed to `shouldPrefetch`. This helps to clarify that it is a boolean, rather a property that returns a type. `prefetchType` is deprecated but not removed. — [3646][]
+-   Store-view specific label and localization support has been added for product attributes. — [17][]
+-   Refactored the reCaptcha to ensure it is backwards compatible with 12.2.0. — [3696][]
+-   The "Sign In" link has been removed from the 2nd stage of checkout. This ensures the cart configuration (guest or account holder) does not change in the middle of checkout. — [3672][]
+-   Images from Page Builder content now include height and width dimensions. This helps reduce CLS when rendering the images. — [files][]
```

Comment:
You've got 2 different links assigned to `[files]` I feel like this could break something?